config/v1/types_scheduling.go

```diff
@@ -20,7 +19,10 @@ type Scheduling struct {
 }
 
 type SchedulingSpec struct {
-	// default node selector (I would be happy to see this die....)
+	// SchedulerConfigMap is a reference to a ConfigMap containing scheduler policy which contains
+	// user specified predicates and priorities.
+	// The namespace for this configmap is openshift-config
+	SchedulerConfigMap ConfigMapNameReference `json:"schedulerConfigMap,omitempty"`
```

Comment:
Since there is such a thing as the "scheduler config" vs "policy config", can we call this `PolicyConfigMap`?  The fact that it is the scheduler policy is implied by being in the scheduler config spec.

Reply:
Makese sense @sjenning. I updated the field to `policyConfigMap`